modules/tile-converter/src/i3s-converter/i3s-converter.ts

```diff
@@ -420,14 +425,40 @@ export default class I3SConverter {
    */
   private async _createAndSaveTileset(outputPath: string, tilesetName: string): Promise<void> {
     const tilesetPath = join(`${outputPath}`, `${tilesetName}`);
+
+    if (await isFileExists(join(tilesetPath, `${tilesetName}${DUMP_FILE_SUFFIX}`))) {
+      const {options, tilesConverted, textureSetDefinitions} = await openJson(
+        tilesetPath,
+        `${tilesetName}${DUMP_FILE_SUFFIX}`
+      );
+      if (ConversionDump.compareOptions(options, this.options) && this.options.inquirer) {
+        const result = await this.options.inquirer.prompt([
+          {
+            name: 'resumeConversion',
+            type: 'confirm',
+            message:
+              'Dump file of the previous conversion exists, do you want to resume that conversion?'
+          }
+        ]);
+        this.resumeConversion = result.resumeConversion;
+        if (this.resumeConversion) {
+          this.conversionDump.init(options, tilesConverted, textureSetDefinitions);
+        }
+      }
+    }
+
+    this.layers0Path = join(tilesetPath, 'SceneServer', 'layers', '0');
+
     // Removing the tilesetPath needed to exclude erroneous files after conversion
     try {
-      await removeDir(tilesetPath);
+      await removeDir(this.resumeConversion ? join(this.layers0Path, 'nodepages') : tilesetPath);
```

Comment:
Shared resources are there in the dump and is not converted/written again if it's already done during previous conversion. JSONs (3DNodeIndexDocuments in case of slpk and index.json if no slpk) will be overwritten.